pipelines/toast_ground_sim.py

```diff
@@ -1310,9 +1315,21 @@ def main():
 
     localpix, localsm, subnpix = get_submaps(args, comm, data)
 
-    # Scan input map
+    if args.input_pysm_model:
 
-    signalname = scan_signal(args, comm, data, counter, localsm, subnpix)
+        op_sim_pysm = tt.OpSimPySM(comm=comm,
+                                   out='signal',
+                                   pysm_model=args.input_pysm_model,
+                                   focalplanes=[s[2] for s in schedules],
+                                   nside=args.nside,
+                                   subnpix=subnpix, localsm=localsm)
+
+        op_sim_pysm.exec(data)
```

Comment:
@tskisner @keskitalo before `data` had just one object `signal`, now instead I have one `signal_detname` per detector. How should modify the copy operations `setup_sigcopy` and `copy_signal`?

Reply:
Those functions have changed quite a bit in my working copy of the ground simulation script. We no longer create persistent CacheCopy operators. The pysm operator just needs to cache the signal under the same name as the old ScanSignal operator did.

Reply:
thanks, the problem is that the old `scan_signal` had only 1 signal, now I have 1 signal per channel. Now I am saving it under `signal_fake_0A`, `signal_fake_0B` and so on. Is it ok?
Also should I rebase on your new version? is it in master?

Reply:
That is the correct naming convention for cache objects. I'm still testing the changes to the pipeline script in my branch. I'll let you know once it is posted.

The layout of the pipeline has changed enough that you may find it easier to insert the operator call directly into the new pipeline rather than rebasing.